plugin/session_buffer.py

```diff
@@ -181,6 +187,7 @@ def add_session_view(self, sv: SessionViewProtocol) -> None:
 
     def remove_session_view(self, sv: SessionViewProtocol) -> None:
         self._clear_semantic_token_regions(sv.view)
+        self.remove_all_inlay_hints()
```

Comment:
That's not right because phantoms are per-buffer so should not remove them if the first view of many is closed, for example. You can probably check if any views are left and only do that if not.

Reply:
[56f4f6e](https://github.com/sublimelsp/LSP/pull/2018/commits/56f4f6e004ce8b50909a954ddaaaffb0f7a5aea7) take a look if this is ok